docs/getting-started/03_build.md

```diff
@@ -6,12 +6,12 @@
    1. nvm
    1. yarn
    1. gulp-cli
-1. Build frontend assets
+1. Build frontend development assets
    1. `nvm use`
-   1. `yarn install:theme`
+   1. `yarn install`
    1. `gulp dist`
-1. Install backend dependencies
-   1. `so composer install`
+1. Install backend development dependencies
+   1. `composer install` or `so composer install` if using SquareOne Global Docker
```

Comment:
Should add a note that their locally installed composer version should match that of the project to ensure all GitHub actions, deploys etc match.